sencha-workspace/SlateTasksTeacher/app/controller/Viewport.js

```diff
@@ -6,8 +6,8 @@ Ext.define('SlateTasksTeacher.controller.Viewport', {
 
 
     views: [
-        'StudentsGrid@Slate.cbl.view.teacher',
-        'GridLegend@Slate.cbl.view.teacher'
+        'StudentsGrid@SlateTasksTeacher.view.main',
```

Comment:
@aliyeysides again these can be shortened now and drop the `main` segment of the namespace